JavaScript/JavaScriptSDK.Tests/CheckinTests/Util.tests.ts

```diff
@@ -5,6 +5,84 @@ class UtilTests extends TestClass {
 
     public registerTests() {
         var Util = Microsoft.ApplicationInsights.Util;
+
```

Comment:
I like these new tests. They are small, easy to understand and test a single concern. :+1: